packages/menu/demo/menu.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, ArgsTable, Canvas, Story, Markdown } from '@storybook/addon-docs';
```

Comment:
I'm not opposed to trying it out. Not sure if it makes sense in this PR since it's on a product timeline but I'm gonna look into it cuz I'm curious. Thanks!

Reply:
For sure, this is a long term effort that's not related to this at all. However I do agree, it's worth looking into at some point.